### ISSUE TITLE
Hope you can add Time support

### ISSUE THREAD

**Issue by camsong** [uxsolutions/bootstrap-datepicker#29]
I can only pick a date, but without hour and minute, hope you can add time support, thanks.

**Comment by jeremyhaile** [comment]
@nicinabox I really like the design of your date/time picker. Unfortunately, you have dependencies on lodash, handlebars, moment, and bootstrap-datepicker. For such a simple widget, this is a ton of dependencies.  We don't currently use lodash and handlebars, and don't want to add them just for a date/time picker.

Any chance you guys are planning to reduce your dependencies?  I think that would allow you to get wider adoption.

**Comment by nicinabox** [comment]
Thanks @jeremyhaile. You're right, lots of dependencies for something fairly simple. All of those dependencies existed already in our application, so the overhead was pretty small for us. If there's interest in trimming the fat I suggest making an issue over there so there's more visibility within the company and I'll see what we can do :)

**Comment by Azaret** [comment]
Time fly.
But this plugin is no dead, I hope not much people lost hope.

@acrobat @hebbet I think this could be a must for some 2.0 release.
[bootstrap-timepicker](http://jdewit.github.io/bootstrap-timepicker/) looks quite pretty and the most completed imo. Also since it is not maintained anymore, if @jdewit agrees we could somehow merge it in.

UI speaking we can keep what we have when we only need date, and keep timepicker's one when only time is needed. When needed both, I think simply appending the timepicker at the bottom should do the trick.

Code-behind speaking, it will need indeed some work, we should see where to place this in the roadmap towards the other hundreds issues.

What do you guys think ?
(Yes now @eternicode has opened this project I'm determined to not let it die :P)

**Comment by acrobat** [comment]
Yes that's definitely a feature we want to have, possibly in a 1.X release if we can finalize this feature! Otherwise it's indeed something for 2.0

**Comment by Azaret** [comment]
I wrote that before getting like page 3 of the issues (there is too many tickets here) so I found the PR already made.

**Comment by Dimas-sc** [comment]
Having the date-range feature with time pickup functionality would be a killer feature. Hope we can see this implemented soon.

**Comment by bennybee** [comment]
+1 to this 
JNajera's looks the easiest to use (least clicks)
Frustrating that the eonasdan version only shows you the selected time once you open the time picker

**Comment by bennybee** [comment]
The UI implementation of Date/Time in the 'Datatables' plugin would be ideal for me:
https://editor.datatables.net/examples/dates/datetime.html
(click a row then 'edit)
